src/main/java/it/pagopa/afm/marketplacebe/service/TouchpointService.java

```diff
@@ -0,0 +1,80 @@
+package it.pagopa.afm.marketplacebe.service;
+
+import it.pagopa.afm.marketplacebe.exception.AppError;
+import it.pagopa.afm.marketplacebe.exception.AppException;
+import it.pagopa.afm.marketplacebe.model.PageInfo;
+import it.pagopa.afm.marketplacebe.model.touchpoint.Touchpoint;
+import it.pagopa.afm.marketplacebe.model.touchpoint.TouchpointRequest;
+import it.pagopa.afm.marketplacebe.model.touchpoint.Touchpoints;
+import it.pagopa.afm.marketplacebe.repository.TouchpointRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Slf4j
+public class TouchpointService {
+
+    @Autowired
+    private TouchpointRepository touchpointRepository;
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+    public Touchpoints getTouchpoints() {
+        List<Touchpoint> touchpointList = new ArrayList<>();
+
+        touchpointRepository
+                .findAll()
+                .forEach(t -> touchpointList.add(modelMapper.map(t, Touchpoint.class)));
```

Comment:
```suggestion
.findAll()
.map(elem-> modelMapper.map(t, Touchpoint.class))
Collectors.toList()
```

Reply:
I'd like to implement this suggestion, but `findAll()` method returns an Iterable, which does not provide a map method

Reply:
Ah ok